pubsub/rabbitpubsub/rabbit_test.go

```diff
@@ -66,6 +66,22 @@ func TestConformance(t *testing.T) {
 	drivertest.RunConformanceTests(t, harnessMaker, asTests)
 }
 
+func BenchmarkRabbit(b *testing.B) {
+	ctx := context.Background()
+	h := &harness{conn: mustDialRabbit(b)}
```

Comment:
Why does this create a harness but the GCP one doesn't?

Reply:
The GCP harness does record/replay, which we don't want for this.